hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/keyvalue/TestKeyValueHandler.java

```diff
@@ -458,4 +485,77 @@ private static ContainerCommandRequestProto createContainerRequest(
         .setContainerID(containerID).setPipelineID(UUID.randomUUID().toString())
         .build();
   }
+
+  @Test
+  public void testReadBlock() throws IOException {
```

Comment:
Could you add add an end to end case, to test read an empty block?

Reply:
Add `testReadEmptyBlock` in the `TestStreamBlockInputStream` to test read an empty block.